biggraphite/cli/command_stats.py

```diff
@@ -115,35 +113,31 @@ def run(self, accessor, opts):
 
         See command.CommandBase.
         """
-        self.ns = Namespaces(opts.conf)
         accessor.connect()
-        accessor.map(
-            self.stats,
-            start_key=opts.start_key, end_key=opts.end_key,
-            shard=opts.shard, nshards=opts.nshards
-        )
+        if accessor.get_accessor_name() == "elasticsearch":
```

Comment:
You could use opts.driver or opts.metadata_driver

Reply:
No, it's not really reliable. I could reimplement a function based on accessor_from_settings but I'm not sure it's simplier